test_scripts/Defects/5_0/1846_Mismatch_for_mandatory_parameter_between_MOBILE_API_and_HMI_API.lua

```diff
@@ -0,0 +1,56 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/SmartDeviceLink/sdl_core/issues/1846 
+--
+-- Description:
+-- Mismatch for mandatory parameter between MOBILE_API and HMI_API
+--
+-- There is a mismatch for the mandatory parameter between the MOBILE_API and the HMI_API for the 
+-- following parameters:
+--     distanceToManeuver
+--     distanceToManeuverScale
+-- The MOBILE_API is reflecting that these parameters are not mandatory, whereas the HMI_API 
+-- reflects that they are mandatory.
+-- 
+-- WARNING: you should include "Navigation-1" group in "sdl_preloaded_pt.json" file before run 
+-- this test!
+---------------------------------------------------------------------------------------------------
```

Comment:
@KrupenichOleg Please update description in one of two ways:
1. In case .... SDL does ...
2. Steps ... Expected ...

Reply:
@dboltovskyi Done in commit [20db55e](https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/2120/commits/20db55e658cf8ba6863901b2f31f3e9d901f7ae0). Also PTU added to preconditions.